examples/directory/BUILD.bazel

```diff
@@ -30,6 +30,9 @@ sh_test(
         "minify",
     ],
     data = ["minify"],
+    env = {
+        "BAZEL_BINDIR": "$(BINDIR)",
```

Comment:
BAZEL_BINDIR? 🤔

Reply:
just coincidence, not used by rules_js actions - gives an easier way for the bash program to locate files.

Reply:
oh I see
```
readonly in_folder=$(rlocation $TEST_WORKSPACE/$(dirname $TEST_BINARY)/$2/$BAZEL_BINDIR)
```

Reply:
maybe it should be using a runfiles lookup instead, I just didn't try very hard :)